src/web_parser/__init__.py

```diff
@@ -0,0 +1,3 @@
+import sys
+sys.path.append('.')
```

Comment:
Doesn't append `.` do the same as `./tests`?

Reply:
I had to use "." to append web_parser because tests need to use web_parser and pytest wasn't adding web_parser to the sys.path for me unless we called 'python -m pytest' which I was trying to avoid to get the VS Code test explorer to work.

Reply:
When you do `.append('.')` doesn't that path automatically include `tests`? Or am I misunderstanding? 

Like when I do `sys.path.append('.')` won't that produce a list of all the possible paths? This is all just a NIT so no big deal if you have a reason for it.

Reply:
I see what you mean, I think I tried that and then added "./tests" because something wasn't working, but it could have been for a different reason. I'm curious so I'm going to try it out and see if its needed or not.

Reply:
It does seem to be needed. Removing "./tests" causes a "module not found" error.

Reply:
My bad, thanks for checking this out!